Mods/SML/Source/SML/Public/Registry/ModContentRegistry.h

```diff
@@ -140,6 +141,8 @@ enum class EGetObtainableItemDescriptorsFlags : uint8
 	Default = None,
 };
 
+ENUM_CLASS_FLAGS(EGetObtainableItemDescriptorsFlags);
```

Comment:
IIRC, I introduced this so that I can easily OR together this enum's values. I didn't split this into a separate commit because it's not a huge deal, but I can do if needed.